ganttproject/src/main/java/net/sourceforge/ganttproject/calendar/CalendarEditorPanel.java

```diff
@@ -316,6 +328,10 @@ protected CalendarEvent getValue(int row) {
         return tableModel.getValue(row);
       }
     };
+
+
+    tableAndActions.addAction(new CalendarAction(tableModel));
```

Comment:
I suggest replacing "Add" button with this new functionality. You can override its behavior in `AbstractTableAndActionsComponent::onAddEvent` at line 305

Reply:
Done